### PR TITLE
Issue #10390: resolve forbiddenapi exclude for LineLengthCheckTest

### PR DESCRIPTION
**Pull request by romani**
just one exclude from Issue #10390

### CODE REVIEW

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/sizes/LineLengthCheckTest.java [hunk 1]
```diff
@@ -99,7 +99,7 @@ public void countUnicodePointsOnce() throws Exception {
             "16: " + getCheckMessage(MSG_KEY, 100, 149),
         };
 
-        verify(checkerConfig, getPath("InputLineLengthUnicodeChars.java"), expected);
+        verifyWithInlineConfigParser(getPath("InputLineLengthUnicodeChars.java"), expected);
```
Comment (pbludov) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/sizes/LineLengthCheckTest.java:102]: Do not need to keep the variable `checkerConfig` in this test?

Comment (romani) [reply src/test/java/com/puppycrawl/tools/checkstyle/checks/sizes/LineLengthCheckTest.java:91]: thanks a lot , fixed.

**Comment by romani** [comment]
@nick-mancuso , do you remember why you added this 
https://github.com/checkstyle/checkstyle/commit/739cf8e48e912fbd8ef633171a5a95e9d36e8a36#diff-5ceb800de033aaf514031a343589e7af89bbecafbc322fa7f1ad1967db495458R108-R110

it works even without special charset.

**Comment by nick-mancuso** [comment]
> @nick-mancuso , do you remember why you added this [739cf8e#diff-5ceb800de033aaf514031a343589e7af89bbecafbc322fa7f1ad1967db495458R108-R110](https://github.com/checkstyle/checkstyle/commit/739cf8e48e912fbd8ef633171a5a95e9d36e8a36#diff-5ceb800de033aaf514031a343589e7af89bbecafbc322fa7f1ad1967db495458R108-R110)
> 
> it works even without special charset.

https://github.com/checkstyle/checkstyle/pull/7845#discussion_r392601952

**Comment by romani** [comment]
@rnveach , please double check if this update will not work.

**Review by pbludov — APPROVED** [review]

**Review by nick-mancuso — APPROVED** [review]

**Review by rnveach — APPROVED** [review]
My local is passing for this change.

**Pull request merged**